specifyweb/businessrules/accessionagent_rules.py

```diff
@@ -5,4 +5,7 @@
 @orm_signal_handler('pre_save', 'Accessionagent')
 def agent_division_must_not_be_null(accessionagent):
     if accessionagent.agent_id is None:
-        raise BusinessRuleException(_("AccessionAgent -> Agent relationship is required."))
```

Comment:
Not required for now, but we are looking into getting rid of back-end localization in favor of having just front-end localization. That way, back end just sends the string key and arguments, whereas the front-end resolves that to a localized string.

Benefits:
- instead of having two localization systems, you have only one
- minor, but back-end responses may be smaller

I began doing that on `l10n-improvements`. After the branches are merged, you can look into doing this if you are interested.

Reply:
Added localization to the backend in 6e5af9f and to the frontend in a291cb3